docs/sections/user_guide/yaml/tags.rst

```diff
@@ -0,0 +1,76 @@
+.. _defining_YAML_tags:
+
+Defining YAML ``tags``
+==========================
+
+Tags are used to denote the type of a YAML node when converting to a Python object. Standard YAML tags are defined at http://yaml.org/type/index.html.
+
+Tags may be implicit:
+
+.. code-block:: yaml
+
+   boolean: true
+   integer: 3
+   float: 3.14
+
+Or explicit:
+
+.. code-block:: yaml
+
+   boolean: !!bool "true"
+   integer: !!int "3"
+   float: !!float "3.14"
+
+Additionally, UW extends the following tags and constructors to support other formats:
+
+UW YAML Tags
+------------
+
+``!float``
+^^^^^^^^^^
+
+Converts the tagged node to a Python ``float`` type.
+
+.. code-block:: yaml
+
+   f2: !float '3.14'
+
+
+``!int``
+^^^^^^^^
+
+Converts the tagged node to a Python ``integer`` type.
+
+.. code-block:: yaml
+
+   cycle_day: !int "{{ cycle.strftime('%d') }}"
+
+
+``!include``
+^^^^^^^^^^^^
+
+Parse the tagged file and include its tags.
+
+.. code-block:: yaml
+
+   salad: !INCLUDE [./fruit_config.yaml]
+
+
+``!list``
+^^^^^^^^^
+
+Converts the tagged node to a Python ``list`` type.
+
+.. code-block:: yaml
+
+   file_names: !list [&gfs_file_names]
+
+
+``!remove``
+^^^^^^^^^^^
+
+Removes the tagged YAML key/value pair.
+
+.. code-block:: yaml
+
+   update_values: !remove
```

Comment:
What would you think about a fuller example, e.g.:

--

Given `input.yaml`
```
e: 2.718
pi: 3.141
```
and `supplemental.yaml`
```
e: !remove
```
```
% uw config realize --input-file input.yaml supplemental.yaml --output-format yaml
pi: 3.141
```

? If you like that, I think the other tags could likewise benefit from fuller examples showing inputs and outputs. The ones that involve only a single file could just read from stdin.

Reply:
Thats true! Honestly I found myself getting overly focused on things like 'is this how I want the title to look' and put it up to break that cycle - it can definitely do for the better examples, working on it!